### DISCUSSION TITLE
FVTT MOD 汉化——纠错贴

### DISCUSSION THREAD

**Discussion by Memorize-Remix** [Memorize-Remix/fvtt_mod_chn#2] [General]
你可以在此指出目前汉化mod文本的错误、缺失或其他，欢迎您的回复。

**Comment by ChenQTYF** [comment]
清单文件地址： https://github.com/13996090016/fvtt_mod_chn/releases/latest/download/module.json
目前失效了

**Reply by Memorize-Remix** [reply]
文件上传出错，现已修复。
然后目前的清单地址改正了，请以后使用如下地址：
https://github.com/Memorize-Remix/fvtt_mod_chn/releases/latest/download/module.json

**Comment by MrHorse055** [comment]
```
"walledtemplates.MeasuredTemplateConfiguration.attach-token.Hint": "连接该测量板，使其与 token 一起移动。",
"walledtemplates.MeasuredTemplateConfiguration.attach-token.Hint": "当 token 旋转时，围绕 token 旋转该测量板。",
```
两处的key重复，第二处应为
`"walledtemplates.MeasuredTemplateConfiguration.rotate-with-token.Hint": "当 token 旋转时，围绕 token 旋转该测量板。",`

**Reply by Memorize-Remix** [reply]
感谢指出，我马上更正

**Comment by yahvk-cuna** [comment]
`CALENDARIA.Calendar.Gregorian.Name` 应该是“格里高利历”，现在翻译成了“格拉里昂历法”
